### PR TITLE
Bug fix show sales not aligned

### LINKED ISSUE #284: Show Sales output not aligned

### PR DESCRIPTION
**Pull request by Cazh1**
Fixes #284

### CODE REVIEW

**Comment by Cazh1** [comment]
Fixes #284

File: src/main/java/seedu/cafectrl/ui/Ui.java [hunk 1]
```diff
@@ -185,6 +185,49 @@ public void showMenuDish(String indexNum, String dishName, String dishPrice) {
...
+    }
+
+    public void formatShowSales(String dishName, int dishQty, String dishPrice) {
+        String leftAlignFormat = "| %-38s | %-12s | %-17s |%n";
+        System.out.format(leftAlignFormat, dishName, dishQty, dishPrice);
+    }
+    public void showSalesCost(String front, String back) {
+        String leftAlignFormat = "| %-53s | %-17s |%n";
+        System.out.format(leftAlignFormat, front, back);
+        //showSalesBottom();
```
Comment (DextheChik3n) [inline src/main/java/seedu/cafectrl/ui/Ui.java:224]: Perhaps remove this commented line if it is not used?

File: src/main/java/seedu/cafectrl/data/Sales.java [hunk 2]
```diff
@@ -63,13 +60,13 @@ public void printSales(Ui ui, Menu menu) {
...
             return;
         }
-
+        ui.showSalesBottom();
         for (int day = 0; day < orderLists.size(); day++) {
             OrderList orderList = orderLists.get(day);
 
-            ui.showToUser("Day " + (day + 1) + ":");
-            ui.showToUser(String.format(HEADER_FORMAT, "Dish Name", "Dish Qty", "Total Cost Price"));
+            ui.showSalesTop(day + 1);
```
Comment (DextheChik3n) [inline src/main/java/seedu/cafectrl/data/Sales.java:67]: perhaps you could replace the "1" to prevent magic number?

Comment (DextheChik3n) [reply src/main/java/seedu/cafectrl/data/Sales.java:67]: you could assign "1" as like int DAY_DISPLAY_OFFSET

**Review by DextheChik3n — COMMENTED** [review]
Added comments relating to coding standards

**Review by DextheChik3n — APPROVED** [review]
All's good now 👍

**Pull request merged**